tempto-examples/bin/run_on_docker.sh

```diff
@@ -24,55 +24,47 @@ function retry() {
 }
 
 function hadoop_master_container(){
-  docker ps --format '{{.Names}}' | grep hadoop-master
+  ${DOCKER_COMPOSE} ps -q hadoop-master
```

Comment:
does it work?
I tested with `docker` and it always returned status code 0. That is why I suggested adding `| grep .` at the end of command.